dags/resources/stages/parse/table_definitions/arrakis/factory_event_PoolCreated.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Can you please create another PR, with just this one file first. This way when we merge this PR , the table factory_event_PoolCreated exists so the other tables don't fail getting built.